Cargo.toml

```diff
@@ -18,6 +18,7 @@ blake2b = ["blake2b-rs"]
 [dependencies]
 cfg-if = "0.1"
 blake2b-rs = { version = "0.1", optional = true }
+numext-fixed-hash = "0.1.6"
```

Comment:
I think we had agreed earlier, that `no_std` feature here does not make any sense, it's best we remove it and we should only leverage C based verifier in smart contract? Has anything changed since then?

Reply:
If we decided to remove `no_std` support, we should actually remove the `std` feature from `features` section.

Reply:
With #16 coming I do suggest we remove `no_std` support. @joii2020 can you make the change per @jjyr's suggestion?

Reply:
OK